docs/guides/fabric_synchronization_guide.md

```diff
@@ -0,0 +1,148 @@
+# Fabric Synchronization Guide
```

Comment:
This document should explain much more what is the topology/goal of the example, with context.

Reply:
Added